src/NuGet.Clients/NuGet.PackageManagement.UI/Options/AddMappingDialog.xaml

```diff
@@ -0,0 +1,89 @@
+<Window x:Class="NuGet.Options.AddMappingDialog"
```

Comment:
Discussed offline that a few properties are missing to give this proper "close" behavior like similar dialogs.